src/controllers/Interactions.ts

```diff
@@ -14,13 +16,17 @@ export async function handleInteraction(buttonInteraction: Interaction): Promise
   const message = buttonInteraction.message;
   if (!(message instanceof Message)) return;
 
-  await buttonInteraction.deferUpdate();
-
   switch (buttonInteraction.customId) {
     case ButtonCustomID.JoinQueue: {
       await message.edit(MessageBuilder.disabledQueueButtons(buttonInteraction));
       const ballchasers = await joinQueue(buttonInteraction.user.id, buttonInteraction.user.username);
-      await message.edit(MessageBuilder.queueMessage(ballchasers));
+
+      if (ballchasers.length == 6) {
```

Comment:
If we move the logic to MessageBuilder I think it will convolute the queueMessage method.  The logic would be compounded because when the queue is full it's a different stage.  The buttons are completely different as well.  

Yes we do a zero length check but that only changes the title and description - it's still a queueMessage.  When there are 6 people in the queue it no longer is a queueMessage.

Just my thoughts.

Reply:
If you want me to switch it up let me know and I can give it a shot.

Reply:
Good point. That makes sense to me. We can keep it here. Still need to use `===`.

Reply:
Fixed in next commit.